### PR TITLE
fix: admin not able to delete events

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6324

the problem is due to space around the names of events.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/modals/event-delete-modal.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ export default class EventDeleteModal extends ModalBase {
   @tracked confirmName = '';
 
   get isNameDifferent() {
-    return this.confirmName !== this.eventName;
+    return this.confirmName !== this.eventName?.trim();
```
Comment (iamareebjamal) [inline app/components/modals/event-delete-modal.js:11]: trim confirm name as well

Comment (maze-runnar) [reply app/components/modals/event-delete-modal.js:11]: ok

**Comment by iamareebjamal** [comment]
Also, don't show popup to admins

**Comment by maze-runnar** [comment]
> Also, don't show popup to admins

means no confirmation is required? direct delete in one click?

**Pull request merged**